homeassistant/components/openai_conversation/config_flow.py

```diff
@@ -63,8 +96,59 @@ async def async_step_user(
             _LOGGER.exception("Unexpected exception")
             errors["base"] = "unknown"
         else:
-            return self.async_create_entry(title="OpenAI Conversation", data=user_input)
+            return self.async_create_entry(
+                title="OpenAI Conversation", data=user_input, options=DEFAULT_OPTIONS
```

Comment:
We should never store the default options. That means you can't change them anymore. Any bug in the template is now stored forever.

Reply:
Am now pulling in default options at request time if options are not set